tests/shared/src/test/scala/org/http4s/DecodeSpec.scala

```diff
@@ -93,32 +93,35 @@ class DecodeSpec extends Http4sSuite {
     decoded == Right("")
   }
 
-  if (Platform.isJvm)
-    test("decode should handle unmappable character") {
-      // https://stackoverflow.com/a/22902806
-      val source = Stream(0x80.toByte, 0x81.toByte)
-      val decoded =
-        source.through(decodeWithCharset[Fallible](JCharset.forName("IBM1098"))).compile.string
-      val Left(_: UnmappableCharacterException) = decoded
-    }
+  test("decode should handle unmappable character") {
+    assume(Platform.isJvm, "this test is for JVM only")
```

Comment:
```suggestion
    assume(Platform.isJvm, "IBM1098 charset is unavailable on JS/Native")
```

Reply:
Wow, thanks for that addition. Really helpful info for further explorers 👍🏻